draft-fossati-tls-attestation.md

```diff
@@ -113,89 +114,38 @@ informative:
-  to be generated in-band during the handshake, fewer inter-protocol
-  requirements must be satisfied (see
-  {{attestation-results-extensions}}).
-
-This specification supports both patterns. In the background check model, any
-attestation technology characteristics, such as evidence encoding format and
-semantics of the evidence contents, are agnostic to the TLS handshake itself.
-Similarly, in the passport model, the characteristics of the attestation results,
-such as encoding format and trust relationships, are agnostic to the TLS
-handshake. Moreover, this specification allows both peers to authenticate
-themselves using remote attestation credentials, and for their attestation
-topologies to be independent of each other.
-
-Several mechanisms for producing and encoding evidence are available, such as:
-
-- the Entity Attestation Token (EAT) {{I-D.ietf-rats-eat}}, 
-- the Trusted Platform Modules (TPMs) {{TPM1.2}} {{TPM2.0}},
-- the Android Key Attestation, and
-- Apple Key Attestation. 
-
-Likewise, there are different encodings available for attestation results. One
-such encoding, AR4SI {{?I-D.ietf-rats-ar4si}}, is being standardized by the RATS
-working group.
-
-However, this document does not specify how different attestation technologies
-should be defined, only that they should all be usable within our framework.
-Attestation mechanisms must be defined by companion specifications.
-
-To give the peer information that the handshake signing key is properly secured,
-the associated attestation result has to be appraised by the peer. This must be
-the case when either of the two remote attestation topologies is used. Hence,
-attestation evidence must contain the security state of both the signing key and
-of the platform hosting it. A Platform Attestation Service issues Platform
-Attestation Tokens (PAT) to prove that the Key Attestation Service has not been
-tampered with. The Key Attestation Service in turn issues Key Attestation Tokens
-(KAT) to prove that the signing key is secure, as described in {{handshake-overview}}. The
-security of the protocol critically depends on the verifiable binding between
-these two logically separate units of evidence.
+Attestation is the process by which an entity produces evidence about itself that another party can use to evaluate the trustworthiness of that entity.
+This document describes a series of protocol extensions to the TLS 1.3 handshake that enables the binding of the TLS authentication key to a remote attestation session.
+As a result, a peer can use "attestation credentials", consisting of compound platform evidence and key attestation, to authenticate itself to its peer during the setup of the TLS channel.
+This enables an attester, such as a confidential workload running in a Trusted Execution Environment (TEE), or an IoT device that is trying to authenticate itself to a network access point, to present a more comprehensive set of security metrics to its peer.
```

Comment:
Since you're reworking this: is this duplication between abstract and intro somewhat expected, using them as two completely unrelated chunks of information?

Reply:
now that we tweaked it, are they sufficiently different?